src/Core/src/Graphics/IGeometry.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Microsoft.Maui.Graphics
+{
+	public interface IGeometry
```

Comment:
We share this interface between .NET MAUI Core and Controls. On each implementation there are specific geometries (using BP or not etc).